src/transformers/generation/configuration_utils.py

```diff
@@ -246,8 +246,9 @@ class GenerationConfig(PushToHubMixin):
 
         num_assistant_tokens_schedule (`str`, *optional*, defaults to `"heuristic"`):
             Defines the schedule at which max assistant tokens shall be changed during inference.
-            - `"_heuristic_`: When all _speculative_ tokens are correct, increase `num_assistant_tokens` by 2 else
-              reduce by 1
+            - `"_heuristic_"`: When all _speculative_ tokens are correct, increase `num_assistant_tokens` by 2 else
```

Comment:
```suggestion
            - `"heuristic"`: When all speculative tokens are correct, increase `num_assistant_tokens` by 2 else
```
(let's remove the `_`)